packages/devtools_app/lib/src/screens/profiler/cpu_profile_controller.dart

```diff
@@ -56,37 +54,40 @@ class CpuProfilerController
   static CpuProfileData emptyAppStartUpProfile = CpuProfileData.empty();
 
   /// The analytics screen id for which this controller is active.
-  final String analyticsScreenId;
+  final String? analyticsScreenId;
 
   /// The store of cached CPU profiles for the currently selected isolate.
   CpuProfileStore get cpuProfileStore =>
       _cpuProfileStoreByIsolateId.putIfAbsent(
-        serviceManager.isolateManager.selectedIsolate.value?.id,
+        serviceManager.isolateManager.selectedIsolate.value?.id ?? '',
         () => CpuProfileStore(),
       );
 
   /// Store of cached CPU profiles for each isolate.
-  final _cpuProfileStoreByIsolateId = <String, CpuProfileStore>{};
+  final Map<String, CpuProfileStore> _cpuProfileStoreByIsolateId =
```

Comment:
Any reason for adding a type on the LHS?

Reply:
removed